packages/api-clients/open-api/catalogApi.yml

```diff
@@ -1625,6 +1625,56 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
+  /eservices/{eServiceId}/signalhub/update:
+    parameters:
+      - $ref: "#/components/parameters/CorrelationIdHeader"
+    post:
+      security:
+        - bearerAuth: []
+      tags:
+        - process
+      summary: Enabling/disabling signalhub service
+      operationId: updateSignalHubFlag
+      parameters:
+        - name: eServiceId
+          in: path
+          description: the eservice id
+          required: true
+          schema:
+            type: string
+            format: uuid
+      requestBody:
+        description: A payload containing boolean which describe if signalhub will be enabled or disabled
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/EServiceSignalhubActivateSeed"
+      responses:
```

Comment:
Missing also 400 bad request error in case body is not valid (thrown automatically by the zodios request body validation)